QMLFile.js

```diff
@@ -21,6 +21,7 @@ var fs = require("fs");
 var path = require("path");
 var log4js = require("log4js");
 var logger = log4js.getLogger("loctool.plugin.QMLFile");
+var ContextResourceString = require("loctool/lib/ContextResourceString.js");
```

Comment:
You cannot depend on the loctool in the plugin for the loctool because that creates a circular dependency. You have to create resources with this.API.newResource()